numpy/polynomial/_polybase.py

```diff
@@ -860,7 +860,8 @@ class domain in NumPy 1.4 and ``None`` in later versions.
         -------
         new_series : series
             A series that represents the least squares fit to the data and
-            has the domain specified in the call.
+            has the domain specified in the call. ``newseries.coef`` will
```

Comment:
Needs to run the convert method to see the coefficients in standard form. So maybe
```
has the domain and window specified in the call. If the coefficients for the unscaled and unshifted
basis polynomials are of interest, do ``new_series.convert().coef`` .
```

Reply:
adopted